### PR TITLE
Add status history section to invoice detail page

### PR DESCRIPTION
**Pull request by sandeepsajan0**
Refs #3474 (2nd point)

image

------------

image

### CODE REVIEW

File: hypha/apply/projects/templates/application_projects/invoice_detail.html [hunk 1]
```diff
@@ -16,12 +16,31 @@ <h5 class="heading heading--no-margin">{% trans "For" %}: {{ object.project.titl
 <div class="wrapper wrapper--sidebar wrapper--outer-space-medium">
     <div class="wrapper--sidebar--inner">
         <div class="card card--solid">
-            <p class="card__text"><b>{% trans "Status" %}:</b> {{ object.get_status_display }}</p>
+            <p class="card__text"><b>{% trans "Invoice Number" %}:</b> {{ object.vendor_document_number }}</p>
```
Comment (theskumar) [inline hypha/apply/projects/templates/application_projects/invoice_detail.html:19]:
```suggestion
            <p class="card__text"><b>{% trans "Invoice No." %}:</b> {{ object.vendor_document_number }}</p>
```

Keep it short!? :) cc: @fourthletter

File: hypha/apply/projects/templates/application_projects/invoice_detail.html [hunk 2]
```diff
@@ -16,12 +16,31 @@ <h5 class="heading heading--no-margin">{% trans "For" %}: {{ object.project.titl
...
             {% is_vendor_setup request as show_vendor_information %}
             {% if show_vendor_information %}
             <p class="card__text"><b>{% trans "Vendor" %}:</b> {{ object.project.vendor.name }}</p>
             {% endif %}
-            <p class="card__text"><b>{% trans "Invoice Number" %}:</b> {{ object.vendor_document_number }}</p>
+            <p class="card__text"><b>{% trans "Lead" %}:</b> {{ object.project.lead }}</p>
+            <p class="card__text"><b>{% trans "Fund" %}:</b> {{ object.project.submission.page }}</p>
+        </div>
+        <div class="card card--solid flex" x-data="{ collapsed: true }">
```
Comment (theskumar) [inline hypha/apply/projects/templates/application_projects/invoice_detail.html:27]: You are missing accessibility and html semantics for the clickable element. Take a look at dropdown web component.

File: hypha/templates/includes/sprites.html [hunk 3]
```diff
@@ -236,10 +236,18 @@
...
 
-    <symbol id="arrow-up" viewBox="0 0 15 15">
-        <path fill-rule="evenodd" d="M8 10a.5.5 0 0 0 .5-.5V3.707l2.146 2.147a.5.5 0 0 0 .708-.708l-3-3a.5.5 0 0 0-.708 0l-3 3a.5.5 0 1 0 .708.708L7.5 3.707V9.5a.5.5 0 0 0 .5.5zm-7 2.5a.5.5 0 0 1 .5-.5h13a.5.5 0 0 1 0 1h-13a.5.5 0 0 1-.5-.5z"/>
+    <symbol id="arrow-up" viewBox="0 0 16 17" fill="none">
+        <path d="M8 16.5L8 2.5" stroke-width="2"/>
+        <path d="M15 8L8 2L1 8" stroke-width="2"/>
     </symbol>
 
+
+    <symbol id="arrow-down" viewBox="0 0 16 17" fill="none">
```
Comment (theskumar) [inline hypha/templates/includes/sprites.html:245]: You can use heroicons for this now.

**Comment by fourthletter** [comment]
Looks really good! Thank you.

**Pull request merged**